### PR TITLE
fix: add translate for dropdown menu

### PR DESCRIPTION
**Pull request by stuarthu**
### SUMMARY
Add translate for dropdown menu

### BEFORE

截屏2020-08-11 15 28 43

### AFTER
截屏2020-08-11 16 18 21

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset/translations/zh/LC_MESSAGES/messages.json [hunk 1]
```diff
@@ -1562,7 +1564,12 @@
       "Changed on": ["改变为"],
       "Saved Queries": ["已保存查询"],
       "Show": ["每页显示"],
-      "entries": ["条"]
+      "entries": ["条"],
+      "Edit chart metadata": ["编辑图表"],
```
Comment (iriswonderland13) [inline superset/translations/zh/LC_MESSAGES/messages.json:1568]:
```suggestion
      "Edit chart metadata": ["编辑图表元数据"],
```

File: superset/translations/zh/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -4113,3 +4119,18 @@ msgstr "周"
 
 #~ msgid "Line Width"
 #~ msgstr "线宽"
+
+#~ msgid "Edit chart metadata"
+#~ msgstr "编辑图表"
```
Comment (iriswonderland13) [inline superset/translations/zh/LC_MESSAGES/messages.po:4124]:
```suggestion
#~ msgstr "编辑图表元数据"
```

**Review by iriswonderland13 — APPROVED** [review]
updated translation of "Edit Chart Metadata" to make it more concrete. Otherwise the translation looks great!

**Review by etr2460 — APPROVED** [review]
lgtm, thanks for the translations!

**Pull request merged**